docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,49 +1,54 @@
 _Please request review from `@zooniverse/frontend` team or an individual member of that team._ 
 
-Package:
-
-Closes # .
+## Package:
 
+## Linked Issue and/or Talk Post:
 
 ## Describe your changes:
 
-<em>Helpful explanations that will make your reviewer happy:</em>
+## How to Review
+_Helpful explanations that will make your reviewer happy:_
 - What Zooniverse project should my reviewer use to review UX?
 - What user actions should my reviewer step through to review this PR?
 - Which storybook stories should be reviewed?
 - Are there plans for follow up PR’s to further fix this bug or develop this feature?
 
-
-# Review Checklist
+# Checklist
+_PR Creator - Please cater the checklist to fit the review needed for your code changes._
+_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._
 
 ## General
-
-- [ ] Are the tests passing locally and on Travis?
-- [ ] Is the documentation up to date?
-
-## Components
-- [ ] Has a storybook story been created or updated?
-- [ ] Is the component accessible? 
-  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
-  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
-  - [ ] Is it passing accessibility checks in the storybook?
-
-## Apps
-
-- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
-- [ ] Does it work on mobile?
-- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?
-
-## Publishing
-
-- [ ] Is the changelog updated?
-- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?
-
-## Post-merging
-
-- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
-- [ ] Is the new feature working or bug now fixed?
-  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
-- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
-  - [ ] Is this approved by our designer?
-- [ ] Is this ready for production deployment?
+- [ ] Tests are passing locally and on Github.
+- [ ] Documentation is up to date. Changelog has been updated if appropriate.
+- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected.
+- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari.
+- [ ] FEM works in a mobile browser.
+
+## General UX
```

Comment:
Accessibility testing should be included here, or should be in its own top level Accessibility section immediately after General UX.